src/components/legend/style.js

```diff
@@ -357,8 +357,30 @@ module.exports = function style(s, gd, legend) {
             var d0 = d[0];
             var w = boundLineWidth(d0.mlw, marker.line, MAX_MARKER_LINE_WIDTH, CST_MARKER_LINE_WIDTH);
 
-            p.style('stroke-width', w + 'px')
-                .call(Color.fill, d0.mc || marker.color);
+            p.style('stroke-width', w + 'px');
+
+            var fillColor = d0.mc || marker.color;
+
+            var getPatternAttr = function(mp, dflt) {
+                if(mp && Array.isArray(mp)) {
```

Comment:
Should we handle `typedArrays` here as well?

Reply:
Fixed in 51614783013bdc48ea9d8076f44a31a8da95b5a5